app/hallebarde/infrastructure/token_repository.py

```diff
@@ -0,0 +1,14 @@
+import boto3
+import hallebarde.config
+
+client = boto3.resource('dynamodb')
```

Comment:
I am not able to run the tests because of this line.

--> `botocore.exceptions.NoRegionError: You must specify a region.`